### ISSUE TITLE
Cross entropy loss 

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#165]
Cross-entropy loss, or log loss, measures the performance of a classification model whose output is a probability value between 0 and 1.
<img width="489" alt="Screen Shot 2022-02-03 at 10 11 47 PM" src="https://user-images.githubusercontent.com/5552371/152470526-0e59d8a1-18eb-4d93-a2cc-3059fa87e1ab.png">
